Algorithms/sieveOfEratosthenes.cpp

```diff
@@ -0,0 +1,41 @@
+//Sieve-of-eratosthenes
+#include<iostream>
+using namespace std;
+
+void primeSieve(int n){
+
+    //initialising a large array with all element as 0
+    int prime[1000]={0};
```

Comment:
Received Stack smashing error while running for n > 1001
![image](https://user-images.githubusercontent.com/78142732/135745778-6d887b08-e424-4686-a398-f7aee7ccb970.png)
![image](https://user-images.githubusercontent.com/78142732/135745783-5729396e-256b-4e94-a84e-472922939d7f.png)